src/reducers/CreateUserReducer.ts

```diff
@@ -33,29 +17,6 @@ export function create(
       return { ...state, createErrorMessage: action.data }
     case 'CLEAR_CREATE_ERROR_MESSAGE':
       return { ...state, createErrorMessage: null }
-    case 'CREATE_UPDATE_USERNAME':
-      return {
-        ...state,
-        username: action.data.username
-      }
-    case 'CREATE_UPDATE_PIN':
-      return {
-        ...state,
-        pin: action.data.pin,
-        pinErrorMessage: action.data.error
-      }
-    case 'AUTH_UPDATE_PASSWORD':
-      return {
-        ...state,
-        password: action.data.password,
-        createPasswordErrorMessage: action.data.error
-      }
-    case 'AUTH_UPDATE_CONFIRM_PASSWORD':
```

Comment:
Please delete the matching `AUTH_UPDATE_CONFIRM_PASSWORD` redux action, which is now unused.